frontend/src/metabase/dashboard/components/DashboardGrid.jsx

```diff
@@ -253,7 +239,16 @@ class DashboardGrid extends Component {
   };
 
   onDashCardRemove(dc) {
-    this.setState({ removeModalDashCard: dc });
+    this.props.removeCardFromDashboard({
+      dashcardId: dc.id,
+    });
+    this.props.addUndo({
+      message: t`Removed card`,
+      undo: true,
+      action: () =>
+        this.props.undoRemoveCardFromDashboard({ dashcardId: dc.id }),
+    });
+    MetabaseAnalytics.trackStructEvent("Dashboard", "Remove Card");
```

Comment:
This method implementation is essentially the same as the one in the deleted `RemoveFromDashboardModal` except there is the additional `addUndo` call